x/launch/keeper/msg_send_request_test.go

```diff
@@ -199,6 +214,82 @@ func TestMsgRequestAddAccount(t *testing.T) {
 			wantID:      1,
 			wantApprove: false,
 		},
+		{
+			name: "should prevent send a new request if sender has no balance",
+			inputState: inputState{
+				noAccount: true,
+				coordinator: profiletypes.Coordinator{
+					CoordinatorID: 5,
+					Address:       coordAddr,
+					Active:        true,
+				},
+				chain: types.Chain{
+					LaunchID:        5,
+					LaunchTriggered: false,
+					IsMainnet:       false,
+					CoordinatorID:   5,
+				},
+				fee: sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(1000))),
+			},
+			msg: *types.NewMsgSendRequest(
+				sample.Address(r),
+				5,
+				types.NewAccountRemoval(sample.Address(r)),
+			),
+			wantApprove: false,
+			err:         sdkerrortypes.ErrInsufficientFunds,
+		},
+		{
+			name: "should prevent send a new request if sender has no balance",
```

Comment:
What difference with one above?

Reply:
In this one, we are setting the fee param to be non-zero

Reply:
ah - i had a duplicate -thanks.

removed